transcom/mymove
# kickback B-18322 Update Requested pickup date

## This PR was made a PR that had a deleted branch associated [click here](https://github.com/transcom/mymove/pull/11888)

## [B-18322](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3a873997)

## Summary

Was kicked back from testing due to incorrect values populating. We both looked at what the correct values should be and I updated the code accordingly.
I took this opportunity to expand the test coverage a little bit as well.

## How to test

1. Make a move with PPM shipment on it (be sure to document the planned move date).
2. As a SC user approve the move.
3. As TOO open the move and navigate to the shipments tab.
4. You should be able to observe changes there and be able to confirm that the data shown matches the data entered during move creation.

Planned move date should be the same date that was selected during creation of the move/shipment.
Actual move date will show only if the move has an actual date entered after creation of the move/shipment.
